docs/api/web-contents-view.md

```diff
@@ -0,0 +1,123 @@
+# WebContentsView
+
+> A View that displays a WebContents.
+
+Process: [Main](../glossary.md#main-process)
+
+This module cannot be used until the `ready` event of the `app`
+module is emitted.
+
+```javascript
+const win = new BaseWindow({ width: 800, height: 400 })
+
+const view1 = new WebContentsView()
+win.contentView.addChildView(view1)
+view1.webContents.loadURL('https://my.app/one')
+view1.setBounds({ x: 0, y: 0, width: 400, height: 400 })
+
+const view2 = new WebContentsView()
+win.contentView.addChildView(view2)
+view2.webContents.loadURL('https://my.app/two')
```

Comment:
`https://slack.com`

Reply:
for consistency with base-window.md examples

Reply:
went with `https://github.com/electron/electron` in both cases to keep Slack branding out of it.